shell/app/modules/cmp/pages/cluster-manage/cluster-list.tsx

```diff
@@ -268,7 +268,7 @@ const ClusterList: React.ForwardRefRenderFunction<{ reload: () => void }, IProps
                 },
                 edit: (record: { meta: object }) => {
                   const { meta } = record;
-                  onEdit(meta);
+                  onEdit({ ...meta });
```

Comment:
what's the root cause of this issue?

Reply:
This object is returned by the back end, defined as unextensible, and then set where it is used.

Reply:
This is where it went wrong, but there is no guarantee that this will be used in the previous code, so we are going to make this object unextensible for now. This deconstruction change first fixes a bug on the cmp.
![image](https://user-images.githubusercontent.com/82502479/151473579-1ad7fa90-047d-4df3-b2bc-3a736a944ffc.png)